source/geometries/Next100FieldCage.cc

```diff
@@ -516,7 +534,7 @@ void Next100FieldCage::BuildELRegion()
   if (elfield_) {
     /// Define EL electric field
     UniformElectricDriftField* el_field = new UniformElectricDriftField();
-    G4double global_el_gap_zpos = el_gap_zpos_ - GetELzCoord();
+    G4double global_el_gap_zpos = el_gap_zpos_ - GetCoordOrigin()[2];
```

Comment:
Same as before. I personally find it more intuitive.

Reply:
Done.